docs/manuals/source/BareosBook/BackupBasics.rst

```diff
@@ -0,0 +1,231 @@
+that daily production operations are not affected by the backup.
+The interval also determines what is the maximum age of a new datum before it is protected
+by a backup.
+
+
+Copying the data
+----------------
+
+Computer **data that needs to be backed up** exists in a lot of different ways today, including:
+  * Files
+  * Metadata (like access rights, ownership ...)
+  * Databases
+  * Virtual Machines
+  * Other
+
+To determine which data needs to be backed up (and which not) is very important.
+Often also there exist multiple ways to back up certain data.
+
+Store the backups in a safe place
+---------------------------------
+
+The **location** where backed up data is stored is very important:
+The closer the backup data is to the original data, the faster usually backup
+and restore operations are. At the same time, the probability that the backup
+data will be lost at the same time as the original data also is higher the
+closer the backup data and the original data are.
+
+A backup of your data on the same local disk that also holds the original data will be very fast.
+However, the probability of losing the backup data together with the original data is extremely high.
+Hardware failure of the disk and/or other parts of the computer, theft or lost of your computer,
+ransomware encrypting your hard disk and other reasons of data loss will affect both the original
+and the backup data.
+
+Manually copying the data to an external hard disk regularly will solve the
+problems described above, however the probability that the backup was not done
+before a data loss is quite high as it is a manual operation. A good option for
+a laptop would be an automatic backup to a s3 cloud storage, so that backup
+will be stored independently of the local hardware and will be accessible
+from everywhere. At the same time the backups still can be made automatically
+which is the basis for a reliable backup.
+
+
+
+Restore the original data
+-------------------------
+Restoring the original data is of course the main goal of all the backup
+operation. Being able to restore the original data is only possible if the data
+integrity is guaranteed. This is usually done by securing the data with
+checksums. As the recovery operation is crucial but usually relatively seldom
+required, it is good practice to regularly test the recovery operation.
+Depending on the recovery objective, the restore speed is also a very important
+parameter that needs to be taken into account.
+
+It is also very important to regularly do restore tests.
+Regular restores will reveal any errors in the data backup process and give the
+administrator the security of being able to carry out a targeted and rapid
+restore when necessary.
+
+Data loss
+---------
+The reasons for data loss are manifold and can basically be classified into the
+three categories Accident, Attack and Force majeure.
+
+Protection against these dangers also becomes more difficult in the order
+indicated.
```

Comment:
it probably makes sense to conclude this section with a pie-chart showing what types of data-loss are how common.